go/api/generic_commands.go

```diff
@@ -30,4 +30,65 @@ type GenericBaseCommands interface {
 	//
 	// [valkey.io]: https://valkey.io/commands/del/
 	Del(keys []string) (Result[int64], error)
+
+	// Unlink (delete) multiple keys from the database. A key is ignored if it does not exist.
+	// This command, similar to Del However, this command does not block the server
+	// Note:
+	//
+	// Parameters:
+	//  keys - One or more keys to unlink.
+	//
+	// Return value:
+	//  Return the number of keys that were unlinked.
+	//
+	// Example:
+	//	result, err := client.Unlink([]string{"key1", "key2", "key3"})
+	//	if err != nil {
+	//	    // handle error
+	//	}
+	//	fmt.Println(result) // Output: int
```

Comment:
`result.Value()` will give output as `int`. Please fix the doc comments for the other commands as well.

Reply:
already corrected in my latest commit. :)